Tests/smokeTest.js

```diff
@@ -0,0 +1,28 @@
+var homepage = require('../Pages/homePage');
+var loginpage = require('../Pages/loginPage');
+describe ('001: Smoke test', function(){
```

Comment:
Add new empty line between describe and var login page

Reply:
I added spaces here